internal/diagnose/errors.go

```diff
@@ -10,3 +10,8 @@ var (
 	ErrPostEvent              = errors.New("there was a failure posting data to New Relic. Please try again later or contact New Relic support. For real-time platform status info visit https://status.newrelic.com/")
 	ErrLicenseKey             = errors.New("the configured license key is invalid for the configured account. Please set a valid license key with the `newrelic profile` command. For more details visit https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/#ingest-license-key")
 )
+
+const PaymentRequiredExceptionMessage = `
+  Your account has exceeded your plan's data limit.
```

Comment:
I might remove one of the "Your/your" to lighten up the message a bit, maybe the 2nd one should be a "the".